provisioning/buffer/aws.sh

```diff
@@ -15,7 +15,19 @@ fi
 # Get credentials to the Amazon Elastic Kubernetes Service
 aws eks update-kubeconfig --name "$AWS_EKS_CLUSTER_NAME" --region "$AWS_REGION"
 
+if [ "${TMP_ETCD_REDEPLOY:-false}" = "true" ]
+then
+    kubectl delete deployment buffer-api -n buffer
+    kubectl delete deployment buffer-worker -n buffer
```

Comment:
Toto by som nerobil, ... pri buffer API nam to nevadi, ale pri Templates API to sposobi vypadok.

Uplne staci zamazat iba etcd.

Reply:
Teda tu to mozes  nechat ak chces.

Reply:
Jo tím jsme si nebyl jistý, viz co jsem psal na slack. Odeberu.